### DISCUSSION TITLE
Google Play release upload error.

### DISCUSSION THREAD

**Discussion by liljeblad** [shogo4405/HaishinKit.kt#144] [Q&A] [answered]
Yesterday, I tried to upload a release of an app that uses this library but the binary upload was rejected with message: 

> APK must not have com.android.graphics.injectLayers.enable set to true in AndroidManifest.xml.

I'm not really familiar with this meta-data value, I simply removed the manifest entry using `<meta-data android:name="com.android.graphics.injectLayers.enable" tools:node="remove" />`.

Is it safe to do this? As far as I can tell the entry is to meant to enable validation layers during development. 
Is this meta-data entry needed in the distribution at all?

**Comment by shogo4405** [comment, marked as answer]
It should be fine as it refers to a verification layer used for debugging purposes. 
https://developer.android.com/ndk/guides/rootless-debug-gles

I think it should work, but I recommend doing a quick test before releasing it. Since it's OpenGL, just confirming that the video is being streamed and playing should be enough.

**Reply by liljeblad** [reply]
Will do, thank you for the blazing fast response!
